cnf-certification-test/networking/icmp/icmp.go

```diff
@@ -43,11 +43,48 @@ type PingResults struct {
 	received    int
 	errors      int
 }
+type RequiredFuncs interface {
+	TestPing(sourceContainerID *provider.Container, targetContainerIP netcommons.ContainerIP, count int) (results PingResults, err error)
+}
+
+type Object bool
```

Comment:
Why use a `bool` type here instead of a `struct`?

Reply:
I could not find something to store in the struct

Reply:
Fixed following Gonzalo's comment